pkg/kubectl/cmd/describe.go

```diff
@@ -64,6 +75,24 @@ func RunDescribe(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []s
 		return err
 	}
 
+	if len(args) == 0 {
+		fmt.Fprint(out, `
```

Comment:
I think it's not an error message, just a hit for user, error message is returned below.